CONTRIBUTING.md

```diff
@@ -1,96 +1,60 @@
 
-Grafana uses GitHub to manage contributions. Contributions take the form of pull requests that will be reviewed by the core team.
+Thanks for your interest in contributing to Grafana! We welcome all people who wants to contribute in a healthy and constructive manner within our community. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](CODE_OF_CONDUCT.md).
 
-- If you are a new contributor, see [Steps to contribute](#steps-to-contribute).
+This document is a guide to help you through the process of contributing to Grafana.
 
-- If you have a trivial fix or improvement, go ahead and create a pull request.
+## Become a contributor
 
-- If you plan to do something more involved, then discuss your idea on the respective [issue](https://github.com/grafana/grafana/issues) or create a [new issue](https://github.com/grafana/grafana/issues/new) if one does not exist. This helps avoid unnecessary work and gives you and us a good deal of inspiration.
+You can contribute to Grafana in several ways. Here are some examples:
 
-- Sign our [CLA](http://docs.grafana.org/contribute/cla/).
+- Contribute to the Grafana codebase.
+- Report and triage bugs.
+- Develop community plugins and dashboards.
+- Write technical documentation and blog posts, for users and contributors.
+- Organize meetups and user groups in your local area.
+- Help others by answering questions about Grafana.
 
-- Follow the code style guides:
-  - [Backend style guide](https://github.com/grafana/grafana/tree/master/pkg)
-  - [Frontend style guide](https://github.com/grafana/grafana/tree/master/style_guides)
+For more ways you can contribute, check out the [Open Source Guides](https://opensource.guide/how-to-contribute/).
 
-## Steps to contribute
+### Report bugs
 
-Should you wish to work on a GitHub issue, check first if it is not already assigned to someone. If it is free, you claim it by commenting on the issue that you want to work on it. This is to prevent duplicated efforts from contributors on the same issue.
+Report a bug by submitting a [bug report](https://github.com/grafana/grafana/issues/new?labels=type%3A+bug&template=1-bug_report.md). Make sure that you provide as much information as possible on how to reproduce the bug.
 
-Please check the [`beginner friendly`](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) and [`help wanted`](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) labels to find issues that are good for getting started. If you have questions about one of the issues, with or without the tag, please comment on them and one of the core team or the original poster will clarify it.
+Before submitting a new issue, try to make sure someone hasn't already reported the problem. Look through the [existing issues](https://github.com/grafana/grafana/issues) for similar issues.
```

Comment:
```suggestion
Before submitting a new issue, ensure is not reported already. Look through the [existing issues](https://github.com/grafana/grafana/issues) for similarities.
```

Correct me, if I'm wrong but we want to don't want it to be optional.

Reply:
It's not, but sometimes it can be difficult to guarantee it hasn't already been reported. I'd prefer to err on the side of accepting here, to avoid missing out on issues. Is there a way a way to rephrase it, that's not as strict?

Reply:
that's a good shout. Sadly, I don't have a good answer for that. Feel free to leave it as is.